MdePkg/Include/IndustryStandard/SmBios.h

```diff
@@ -1524,7 +1524,7 @@ typedef struct {
   UINT8    AsyncSurpriseRemoval    : 1;
   UINT8    FlexbusSlotCxl10Capable : 1;
   UINT8    FlexbusSlotCxl20Capable : 1;
-  UINT8    Reserved                : 1; ///< Set to 0.
+  UINT8    FlexbusSlotCxl30Capable : 1; ///SMBIOS spec 3.7.0 updated CXL 3.0 support

```

Comment:
Previous additions don't mention when they were added, so the comment feels superfluous here.